src/components/project/ProjectCollectionBlock.tsx

```diff
@@ -193,7 +193,7 @@ const useProjectPreviewCollection = (
                 ...condition,
                 ...(Object.keys(Array.isArray(orderBy) ? orderBy[0] : orderBy)[0] === 'position'
                   ? { position: { _gt: data?.project.slice(-1)[0]?.position } }
-                  : { created_at: { _lt: data?.project.slice(-1)[0]?.created_at } }),
+                  : { published_at: { _lt: data?.project.slice(-1)[0]?.published_at } }),
```

Comment:
line 155 
must also change to published_at

Reply:
line 153 had modified
line 155 dose't need to modify. I had modify orderBy in tabContents where in published